ui/src/pages/backoffice/Series/SeriesDetails/SeriesDetails.js

```diff
@@ -41,10 +41,18 @@ export default class SeriesDetails extends Component {
 
   seriesPanels = () => {
     const { data } = this.props;
+    let literatureNum = 0
+    if (data.metadata && data.metadata.relations_metadata) {
+      literatureNum = data.metadata.relations_metadata.serial ?
+        data.metadata.relations_metadata.serial.length :
+        data.metadata.relations_metadata.multipart_monograph.length
+    } else {
+      literatureNum = 0
+    }
     const panes = [
       {
         key: 'series-documents',
-        title: 'Literature in this series',
+        title: 'Literature in this series (' + literatureNum + ')',
```

Comment:
This is the new madness in string interpolation :)
```suggestion
        title: `Literature in this series (${literatureNum})`,
```